bigbluebutton-html5/imports/ui/components/nav-bar/container.jsx

```diff
@@ -39,17 +37,11 @@ export default withTracker(() => {
   }
 
   const checkUnreadMessages = () => {
-    const users = userListService.getUsers();
-
-    // 1.map every user id
-    // 2.filter the user except the current user from the user array
-    // 3.add the public chat to the array
-    // 4.check current user has unread messages or not.
-    return users
-      .map(user => user.id)
-      .filter(userID => userID !== Auth.userID)
-      .concat(PUBLIC_GROUP_CHAT_ID)
-      .some(receiverID => ChatService.hasUnreadMessages(receiverID));
+    const activeChats = userListService.getActiveChats();
+    const hasUnreadMessages = activeChats
```

Comment:
Why not return the result directly?

Reply:
IMO, is good as-is, the variable has a meaningful name explaining his content avoiding the next programmer pass function by function to know the content